### DISCUSSION TITLE
mod tier for searching item

### DISCUSSION THREAD

**Discussion by carcarhin** [SnosMe/awakened-poe-trade#576] [Q&A] [answered]
like a boot
+(70–79) to maximum Life is T2 life
+(80–89) to maximum Life is T1 life
if my boot hv 85 life
if direct search 85 life, the price may not be accuracy, boot with 80-84 will not shown
cause someone can use divine orb to get the maximum life if other boot hv 80 life, that also T1 life 
can awakened poe trade auto set the searching range to the lowest number of that tier?
many thanks

**Comment by SnosMe** [comment, marked as answer]
The default settings do exactly what you've described, but you likely changed it to "Exact roll"

![image](https://user-images.githubusercontent.com/4292308/159120305-b769d70f-8b48-4609-a8e1-6d6931758a51.png)
